src/app/Console/Commands/Install.php

```diff
@@ -26,45 +31,191 @@ class Install extends Command
      */
     protected $description = 'Install Backpack requirements on dev, publish files and create uploads directory.';
 
+    /**
+     * Addons variable.
+     *
+     * @var array
+     */
+    protected $addons = [
+        Addons\RequirePro::class,
+        Addons\RequireDevTools::class,
+        Addons\RequireEditableColumns::class,
+    ];
+
     /**
      * Execute the console command.
      *
      * @return mixed Command-line output
      */
     public function handle()
     {
-        $this->progressBar = $this->output->createProgressBar(5);
-        $this->progressBar->minSecondsBetweenRedraws(0);
-        $this->progressBar->maxSecondsBetweenRedraws(120);
-        $this->progressBar->setRedrawFrequency(1);
-
-        $this->progressBar->start();
-
-        $this->info(' Backpack installation started. Please wait...');
-        $this->progressBar->advance();
+        $this->infoBlock('Installing Backpack CRUD:', 'Step 1');
 
-        $this->line(' Publishing configs, views, js and css files');
+        // Publish files
+        $this->progressBlock('Publishing configs, views, js and css files');
         $this->executeArtisanProcess('vendor:publish', [
-            '--provider' => 'Backpack\CRUD\BackpackServiceProvider',
+            '--provider' => BackpackServiceProvider::class,
             '--tag' => 'minimum',
         ]);
+        $this->closeProgressBlock();
 
-        $this->line(" Creating users table (using Laravel's default migration)");
+        // Create users table
+        $this->progressBlock('Creating users table');
         $this->executeArtisanProcess('migrate', $this->option('no-interaction') ? ['--no-interaction' => true] : []);
+        $this->closeProgressBlock();
 
-        $this->line(" Creating App\Http\Middleware\CheckIfAdmin.php");
+        // Create CheckIfAdmin middleware
+        $this->progressBlock('Creating CheckIfAdmin middleware');
         $this->executeArtisanProcess('backpack:publish-middleware');
+        $this->closeProgressBlock();
+
+        // Install Backpack Generators
+        $this->progressBlock('Installing Backpack Generators');
+        $process = new Process(['composer', 'require', '--dev', 'backpack/generators']);
+        $process->setTimeout(300);
+        $process->run();
+        $this->closeProgressBlock();
+
+        // Create users
+        $this->createUsers();
+
+        // Addons
+        $this->installAddons();
+
+        // Done
+        $url = Str::of(config('app.url'))->finish('/')->append('admin/');
+        $this->infoBlock('Backpack installation complete.', 'done');
+        $this->note("Go to <fg=blue>$url</> to access your new admin panel.");
+        $this->note('You may need to run `php artisan serve` to serve your Laravel project.');
```

Comment:
using the backticks here does not make `php artisan serve` standout. We should probably add some color/bold whatever to it.

Reply:
I agree 👌
<img width="470" alt="image" src="https://user-images.githubusercontent.com/1838187/182359144-04edf78e-e1e1-42cb-8c7f-e1fed5361a1a.png">

That one was easy 😅